docs/ldml/tr35-keyboards.md

```diff
@@ -1022,12 +1021,33 @@ See  [`<displayOptions baseCharacter=…/>`](#element-displayoptions).
 
 ### Element: display
 
-The `display` element describes how a character, that has come from a `keys/key` element, should be displayed on a keyboard layout where such display is possible.
+The display elements can be used to describe what is to be displayed on the keytops for various keys. For the most part, such explicit information is unnecessary since the `@to` element from the `keys/key` element will be used for keytop display. However some characters, such as diacritics, do not display well on their own.
+
+Another useful scenario is where there are doubled diacritics, or multiple characters with spacing issues.
```

Comment:
Minor; a list of single line paragraphs is strange. Do you mean this to be a list?